cpp/include/cuml/linear_model/glm.hpp

```diff
@@ -104,6 +104,7 @@ void gemmPredict(const raft::handle_t &handle, const double *input, int n_rows,
  * scikit, the bias will not be regularized.
  * @param max_iter              limit on iteration number
  * @param grad_tol              tolerance for gradient norm convergence check
+ * @param change_tol            tolerance for function change convergence check
```

Comment:
It'd be good to add the same explanation that was added in the .pyx here

Reply:
I added a somewhat shorter explanation (omitted the comparison to sklearn). I hope this is enough :)